mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -1090,6 +1090,20 @@ func ResourceContainerCluster() *schema.Resource {
 			},
 <% end -%>
 
+			"security_posture": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
```

Comment:
I wonder what is the reason to set `Computed: true` here? `Computed:     true` is used in the case that the API will returns default value if no value is specified in the request.

Reply:
Good question! I couldn't pass the Acceptance test locally if I don't set the `Computed:true`. Although, it didn't return error when I did manual test without `Computed:true`. I really could use your advice on this. 

Could it be the configurations I have in my local development environment?

Reply:
Did you get any error in the logs when running the acceptance test locally without `Computed:true`?

Reply:
Yes.

Error log like below,
```
error=
  | After applying this test step, the plan was not empty.
  | stdout:
  | 
  | 
  | Terraform used the selected providers to generate the following execution
  | plan. Resource actions are indicated with the following symbols:
  |   ~ update in-place
  | 
  | Terraform will perform the following actions:
  | 
  |   # google_container_cluster.with_security_posture_config will be updated in-place
  |   ~ resource "google_container_cluster" "with_security_posture_config" {
  |         id                              = "projects/wangbei-gke-dev/locations/us-central1-c/clusters/tf-test-cluster-6mwho946jg"
  |         name                            = "tf-test-cluster-6mwho946jg"
  |       - workload_vulnerability_scanning = "DISABLED" -> null
  |         # (25 unchanged attributes hidden)
  | 
  |         # (16 unchanged blocks hidden)
  |     }
  | 
  | Plan: 0 to add, 1 to change, 0 to destroy.
```

Reply:
I think the `Computed: true` is needed.